docs/api-reference/io-columns-anomaly-detection.md

```diff
@@ -0,0 +1,6 @@
+### Input and Output Columns
+The input features column data must be a known-sized vector of <xref:System.Single>. This trainer outputs the following columns:
+
+| Output Column Name | Column Type | Description|
+| -- | -- | -- |
+| `Score` | <xref:System.Single> | The score that was calculated by the anomaly detection model.|
```

Comment:
>  [](start = 38, length = 1)

if the range is (-inf,+inf) we call them 'unbounded score' to be clear that it's not normalized or anything like that.